coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
+    aspects = defaultdict(lambda: [])
```

Comment:
Yeah I was just thinking of explicitly defining `aspects['detect'] = []` to maintain consistency of `type(aspects)`.